core/targets.py

```
@@ -1,19 +1,29 @@
-"""
-WHO-Based Nutrition Targets System
-
-RU: Система расчёта целевых значений нутриентов на основе рекомендаций ВОЗ.
-EN: WHO-based nutrition targets calculation system.
+"""WHO-Based Nutrition Targets System.
 
 This module defines the core data structures for user profiles and nutrition targets
 based on WHO/EFSA/DRI recommendations for macronutrients, micronutrients, hydration,
 and physical activity guidelines.
+
+RU: Система расчёта целевых значений нутриентов на основе рекомендаций ВОЗ.
```

Ruff (0.13.3)

Docstring contains ambiguous `В` (CYRILLIC CAPITAL LETTER VE). Did you mean `B` (LATIN CAPITAL LETTER B)?

(RUF002)